static/js/tools/scatter/draw_scatter.ts

```diff
@@ -110,6 +119,57 @@ function addXLabel(
   return xAxisHeight;
 }
 
+/**
+ * Get a d3 scale given the type of scale, domain, and range
+ * @param scaleType type of scale to return
+ * @param domainExtent the min and max of the domain
+ * @param rangeExtent the min and max of the range
+ */
+function getScale(
+  scaleType: ScaleType,
+  domainExtent: [number, number],
+  rangeExtent: [number, number]
+):
+  | d3.ScaleSymLog<number, number>
```

Comment:
could we create a new type that encompasses this? especially since it's repeated throughout this file

Reply:
done